imports/util/email-invite.js

```diff
@@ -1,17 +1,38 @@
 import { Email } from 'meteor/email'
 import url from 'url'
+import bugzillaApi from '../util/bugzilla-api'
+const { callAPI } = bugzillaApi
 
-export const invite = (user, invitedBy) =>
-  Email.send({
-    to: user.emails[0].address,
-    from: process.env.FROM_EMAIL,
-    subject: `Invite by ${invitedBy.profile.name || invitedBy.emails[0].address}`,
-    text: `Hi there,
+export const invite = (user, invitedBy) => {
+  try {
+    const caseId = user.invitedToCases[0].caseId
+    const unitId = user.invitedToCases[0].unitId
```

Comment:
Can also work as 
`const { caseId, unitId, role: inviteeRole } = user.invitedToCases[0]`

Reply:
and to keep it consistent, you should add `accessToken` too, instead of accessing it inline the email body template.